src/main/java/domain/LottoStatistics.java

```diff
@@ -0,0 +1,57 @@
+package domain;
+
+import text.Phrase;
+
+import java.math.BigInteger;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class LottoStatistics {
+    private final HashMap<LottoRank, BigInteger> rankNumbers;
+
+    public LottoStatistics(HashMap<LottoRank, BigInteger> rankNumbers) {
+        this.rankNumbers = rankNumbers;
+    }
+
+    private static String entryToString(Map.Entry<LottoRank, BigInteger> entry) {
+        LottoRank lottoRank = entry.getKey();
+        BigInteger value = entry.getValue();
+        if (lottoRank.getDescription().isEmpty()) {
+            return "";
+        }
+        return Phrase.getLottoRewardStatement(lottoRank.getDescription(),
+                lottoRank.getReward(), value);
+    }
+
+    public HashMap<LottoRank, BigInteger> getRankNumbers() {
+        return rankNumbers;
+    }
+
+    public BigInteger calculateTotalReward() {
+        BigInteger totalReward = new BigInteger("0");
+
+        for (LottoRank lottoRank : rankNumbers.keySet()) {
+            totalReward = totalReward.add(
+                    lottoRank.getReward()
+                            .multiply(rankNumbers.get(lottoRank)));
+        }
+        return totalReward;
```

Comment:
리워드의 총합을 구하고자 한다면 reduce 기능을 이용해도 좋을 것 같아요. 
```suggestion
        return rankNumbers.keySet()
                .stream()
                .map(reward -> reward.getReward().multiply(rankNumbers.get(reward)))
                .reduce(BigInteger.valueOf(0), BigInteger::add);

Reply:
반영되었습니다 👍